test/toolTests.ts

```diff
@@ -1,20 +1,19 @@
-/// <reference path="../typings/index.d.ts" />
-/// <reference path="../_build/tool.d.ts" />
-
 import assert = require('assert');
 import path = require('path');
 import fs = require('fs');
 import shell = require('shelljs');
 import os = require('os');
+
+import * as mocha from 'mocha';
```

Comment:
Would `import * from 'mocha';` allow the syntax checking to succeed without adding `mocha.` before each mocha function? If not, would `require('mocha');` avoid having to add `mocha.` everywhere (I'm not so much worried about this repo, but thinking about when we apply the same pattern from here to the vsts-tasks repo).

fyi - you added a newline above this line that seems to be unintentional